### PR TITLE
Basic Questionnaires Page, User Registration, User Verification and Profile Page

### PR DESCRIPTION
**Pull request by Subritt**
### Features Worked on
- Quaternaries Page
- User Registration
- User Verification
- Profile Model
- Cleaned `__pycache__` folder

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Django #
+*.log
+local_settings.py
+db.sqlite3
+db.sqlite3-journal
+*.pot
+*.pyc
+__pycache__/
+media
```
Comment (swojeet) [inline .gitignore:9]: Let's add a new line at the end of every file that shows "No newline at EOF" symbol.

Comment (Subritt) [reply .gitignore:9]: okay, reflected in `questionnaires` branch

**Comment by swojeet** [comment]
Let's Remove *.pyc files as well

**Comment by swojeet** [comment]
Let's update the PR Title to something meaningful that is relevant to the feature

**Comment by Subritt** [comment]
[Signup](https://trello.com/c/WPKRMr7W/17-signup)

**Comment by Subritt** [comment]
[user verification](https://trello.com/c/kQC2Nxc0/16-user-verification)

**Comment by Subritt** [comment]
[Profile Page](https://trello.com/c/MT6sun9d/22-profile-page)

**Comment by Subritt** [comment]
[Questionnaires and Reply](https://trello.com/c/WzcFeEKc/21-questionnaires-and-reply)

**Comment by Subritt** [comment]
> Let's Remove *.pyc files as well

Done in `questionnaires` branch

**Review by swojeet — APPROVED** [review]

**Pull request merged**